IfcCommunicationsAppliance Rail Import BIMQ 2021-07-21T07:21:13.7228368Z (#177)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayBalise/Documentation.md

> Properties in this property set are applicable for the transponder type of communications appliance, indicated that the transponder is a railway balise.

Reviewer: ```suggestion
Properties  applicable to a railway balise. This property set is applied to a type or occurrence of _IfcCommunicationsAppliance_ with predefined type of TRANSPONDER.
```

> Properties  applicable to a railway balise. This property set is applied to a type or occurrence of _IfcCommunicationsAppliance_ with predefined type of TRANSPONDER.

Reviewer 2: ```suggestion
Properties  applicable to a railway balise. This property set is applied to a type or occurrence of _IfcCommunicationsAppliance_ with predefined type TRANSPONDER.
```
